packages/peregrine/.npmignore

```diff
@@ -1,4 +1,3 @@
 .eslintrc.js
-.storybook
 /lib/**/__{docs,helpers,mocks,stories,tests}__/**
```

Comment:
You could remove `stories` from this line too. It won't break anything keeping it in, so not a huge deal.

Reply:
```suggestion
/lib/**/__{docs,helpers,mocks,tests}__/**
```